### PR TITLE
Upgrade to Helm v3 Beta 4

### LINKED ISSUE #64: Upgrade to Helm v3 Beta 4

### PR DESCRIPTION
**Pull request by hickeyma**
Closes #64

### CODE REVIEW

**Comment by hickeyma** [comment]
This is currently blocked on https://github.com/helm/helm-www/issues/300 as `helm.sh` needs to be able to handle the redirect to `helm.sh/helm/v3`.

**Comment by rimusz** [comment]
helm/helm-www#300  was merged

File: go.mod [hunk 1]
```diff
@@ -22,7 +22,7 @@ require (
 	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f // indirect
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xeipuuv/gojsonschema v1.1.0 // indirect
-	helm.sh/helm v3.0.0-beta.3+incompatible
+	helm.sh/helm/v3 v3.0.0-beta.4+incompatible
```
Comment (rimusz) [inline go.mod]: `+incompatible` is not needed

**Comment by hickeyma** [comment]
> helm/helm-www#300 was merged

Sure is. Working on it @rimusz

File: go.mod (hunk 1 above)
Comment (hickeyma) [reply go.mod]: yes, will be tidying up when push again

Comment (hickeyma) [reply go.mod]: fixed.

**Comment by hickeyma** [comment]
Ready for review.

**Pull request merged**